R/vertica.udx.R

```diff
@@ -0,0 +1,104 @@
+
+
+generate_sql_query <- function(fun, tbl, col_types, col_names, lengths, partition_cols, order_cols, libraries = NULL, ...)
+{
+	
+	params = list(...)
+	params$vertica.dplyr.libraries = libraries
+	params$vertica.dplyr.func = fun
+
+	params = serialize_object(params)
+
+	outcols = list(lengths = lengths, col_types = col_types, names = col_names)
+	outcols = serialize_object(outcols)
+
+	parameters_clause = paste0("using parameters params = '",
+		params, "', outcols='",outcols,"'")
+
+	over_clause = ""
+	
+	source_query <- tbl$query$sql
+	source_query <- paste0("( ", source_query ,") AS ZZZ1")
+
+	query = paste0("select vertica_function_executor(* ", 
```

Comment:
I tried doing it with select but there were problems with strings. it would automatically escape " and ' and do other string related things.